views/pages/resources/mental-wellness.pug

```diff
@@ -0,0 +1,27 @@
+doctype html
+html(lang="en")
+    head 
+        title Mental Wellness &#8211; MathSoc
+        include /views/partials/header.pug
+        link(href="/assets/css/pages/wellness.css" rel="stylesheet")
+
+    body
+        include /views/partials/navbar.pug
+        div(class="content")
+            section(class="white")
```

Comment:
Could we divert from the original design and add a title to this page? (Maybe with a banner image too if you think it would improve the appearance). It would help to fill out the page nicely!

If so, we could make that title `h1` and the on-campus/off-campus headings `h2`, which again might improve on the original wordpress site's design.

Reply:
done :)